docs/website/docs/dlt-ecosystem/verified-sources/rest_api.md

```diff
@@ -576,3 +576,18 @@ See the [incremental loading](../../general-usage/incremental-loading.md#increme
 - `root_key` (bool): Enables merging on all resources by propagating root foreign key to child tables. This option is most useful if you plan to change write disposition of a resource to disable/enable merge. Defaults to False.
 - `schema_contract`: Schema contract settings that will be applied to this resource.
 - `spec`: A specification of configuration and secret values required by the source.
+
+## Troubleshooting
+
+If you encounter any issues while setting up or running the pipeline, check the following:
+
+- Make sure you have the correct access token and other credentials in the `secrets.toml` file.
+- Check the API documentation for the correct endpoint paths and query parameters.
+- Verify the configuration settings in the `rest_api_pipeline.py` file.
```

Comment:
This does not help. You can explain what DictValidationError is and how to troubleshoot it